ilastik/shell/gui/ilastikShell.py

```diff
@@ -160,9 +151,88 @@ def setMemoryBytes(self, bytes):
         self.label.setText("Cached Data: %1.1f MB" % (bytes / (1024.0**2.0)))
 
 
-# ===----------------------------------------------------------------------------------------------------------------===
-# === ProgressDisplayManager                                                                                         ===
-# ===----------------------------------------------------------------------------------------------------------------===
+class LogWindow(QTextEdit):
+    """Window/Widget that shows the log history"""
+
+    def __init__(self, data, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._setup_ui(data)
+
+    def _jump_to_end(self):
+        self.verticalScrollBar().setValue(self.verticalScrollBar().maximum())
+
+    def _setup_ui(self, data):
+        self.setWindowTitle("Log")
```

Comment:
I can foresee scenarios where users discover this feature, but never the "real" log. Then we ask them to provide logs on the forum and they think we're asking them for the contents of this box...

I'd call it "Notifications" (see below)

Reply:
Oops, I think the window title slipped through still saying "Log" 😅

Reply:
yes yes I realized directly after clicking merge  ;).